test/DynamoCoreTests/Updates/UpdatesTests.cs

```diff
@@ -47,5 +51,88 @@ public void UpdateManagerConfiguration_LoadTest()
             Assert.IsNull(UpdateManagerConfiguration.Load("filePath", updateManager));
             Assert.IsNull(UpdateManagerConfiguration.Load(Path.GetTempFileName(), updateManager));
         }
+
+        [Test]
+        [Category("UnitTests")]
+        public void UpdateManagerConfiguration_SaveTest()
```

Comment:
Please dont use underscore, I think out team are trying to get rid of this coding style recently

Reply:
Removed the underscore